workflow/cron/operator.go

```diff
@@ -191,6 +191,10 @@ func (woc *cronWfOperationCtx) terminateOutstandingWorkflows(ctx context.Context
 		woc.log.Infof("stopping '%s'", wfObjectRef.Name)
 		err := util.TerminateWorkflow(ctx, woc.wfClient, wfObjectRef.Name)
 		if err != nil {
+			if errors.IsNotFound(err) {
+				woc.log.Warnf("%s workflow not found", wfObjectRef.Name)
```

Comment:
A more descriptive log will make it easier to pinpoint if it comes up in the future
```suggestion
				woc.log.Warnf("workflow '%s' not found when trying to terminate outstanding workflows", wfObjectRef.Name)
```